src/frontend/scss/colors/_theme.scss

```
@@ -466,6 +472,10 @@ $r-theme: (
     title-color: r-color('charcoal'),
     base-color: r-color('purplish-grey'),
   ),
+  sale-tunnel: (
+    separator-color: r-color('grey87'),
+    email-description-color: r-color('purplish-grey'),
+  ),
```

I'll nest `description` and `block-title` scheme into `sale-tunnel`. IMO, there too generic to be scheme.

```suggestion
  sale-tunnel: (
    separator-color: r-color('grey87'),
    email-description-color: r-color('purplish-grey'),
    description-color: r-color('charcoal'),
    title-color: r-color('charcoal')
  ),
```